azkaban-webserver/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -338,6 +340,24 @@ private void handleExecutionFlowPage(HttpServletRequest req,
       page.render();
       return;
     }
+    
+    Props props = getApplication().getServerProps(); 
+    String execExternalLinkURL = 
+        props.getString("execution.external.link.url", "");
+    String execExternalLinkLabel = 
+        props.getString("execution.external.link.label", "External Analyzer");
+        
+    if (StringUtils.isNotEmpty(execExternalLinkURL)) {
+      StringBuilder builder = new StringBuilder();
+      builder.append(req.getRequestURL());
+      builder.append("?");
+      builder.append(req.getQueryString());
+      String flowExecutionURL = builder.toString();
```

Comment:
I see.

In this case, let's not complicate this feature. 
Support the complete URL only is fine with me. Could you add a comment explaining why the complete URL is used? I don't expect this feature to be widely used other than integration with Dr. Elephant in the near future. 
I agree %url seems to reflect better what's happening than %s.